accepted/2020/linking-libraries.md

```diff
@@ -0,0 +1,152 @@
+# Linking the .NET Libraries
+
+When publishing a self-contained application, the .NET Core runtime is bundled together with the application. This bundling adds a significant amount of content to the packaged application. Depending on the complexity of the application, only a subset of the runtime is required to run the application. These unused parts of the runtime are unnecessary and can be trimmed from the packaged application.
+
+This is especially important for applications that need to be downloaded over the internet, for example Xamarin and Blazor applications. End users on a slow or limited network will be less likely to install and use applications that are too large.
+
+The [ILLinker](https://github.com/mono/linker) is a tool that is able to take an application and trim the classes and methods that are not used by the application. In many scenarios, this results in a significantly smaller packaged application.
+
+In .NET 5 we will modify the [.NET Libraries](https://github.com/dotnet/runtime/tree/master/src/libraries) to work well with the ILLinker. With .NET 5, our goal is to use the .NET Libraries for all .NET applications. In previous releases, Xamarin and Blazor applications have been using the Mono libraries, which have been written with size in mind. However, the .NET Libraries have traditionally been concerned with throughput and not size.
+
+In the context of trimming applications, there are two main concerns:
+
+1. Any code that is used by the application must be preserved in the application.
+2. Any code that is not used by the application can be removed from the application.
+
+Note the difference in "must" and "can" between those two concerns. An application that works is preferred over a smaller application that doesn't. Therefore it is critical that any necessary code must be preserved in the application.
+
+## Scenarios and User Experience
+
+Developers can run the ILLinker on their .NET application and be confident that the resulting application works correctly, while meeting size goals.
```

Comment:
> this now sounds like the ILLinker will be additional work for developers

It can be, right? For example if a developer is using reflection in their application, they may need to add annotations to their code.

I can reword this sentence to:

> The ILLinker can be executed on a .NET application and the developer will be confident that the trimmed application works correctly, while reducing its final size.